### ISSUE TITLE
More consistent colour override behaviour

### ISSUE THREAD

**Issue by CocaineJohnsson** [DeaDBeeF-Player/deadbeef#2316]
Deadbeef version: 1.8.2-1
OS: Arch Linux

More of a nitpick/enhancement request than anything -- though it should be trivial enough

Spectrum analyzer, and highlighted entries (e.g in playlist) should respect `Preferences->Appearance->Override Seekbar/Volumebar/EQ` 'Bars' colour, or a separate entry for overriding these misc. elements should be added as there is currently no way to change these from blue.

**Comment by dmidecodes** [comment]
good point, perhaps even consider playback controls to be affected, too?

will give the entire design a uniform look.
